tokens.xml

```diff
@@ -8087,6 +8230,11 @@ This creature can't be enchanted.</text>
                 <maintype>Artifact</maintype>
                 <cmc>0</cmc>
             </prop>
+            <set picURL="https://media.wizards.com/2022/snc/en_hLVcbOyxF3.png">SNC</set>
+            <set picURL="https://media.wizards.com/2022/snc/en_XHQoWUdbKk.png">SNC</set>
+            <set picURL="https://media.wizards.com/2022/snc/en_u1BnldwqLg.png">SNC</set>
+            <set picURL="https://media.wizards.com/2022/snc/en_5j7mwsCJD8.png">SNC</set>
+            <set picURL="https://media.wizards.com/2022/snc/en_AXaJsB76gI.png">SNC</set>
```

Comment:
there still isn't any use adding multiple copies of a set tag

Reply:
Yes, the only use for this is for someone to pull their favourite token URL out of the file and make it custom, and that's pretty unlikely.
But, you as a reviewer can say which one you'd like to keep. ;)

Reply:
I would like to keep them. That's what we do for other variants within the same set as well.
No harm in keeping them, and yes, some users like the additional info when they want to change it.

Reply:
Alright, but which one do the two of you like most? Which one gets to see the light of day? 😁

Reply:
Isn't that the honor of the person putting the work in to add them to decide? 😉

My favourites are nr. 1 and nr. 5 in your list.

Reply:
it picks the last one and I thought it was pretty good, so it stays